canvas_modules/common-canvas/src/common-properties/util/property-utils.js

```diff
@@ -573,6 +573,18 @@ function convertValueDataTypes(currentParameters, paramDefs) {
 	return convertedCurrentParameters;
 }
 
+// Return string value for {name: string, row: integer, col: integer}
+function getPropertyIdOfTableControlString(propertyId) {
```

Comment:
Note that we do support nesting in properties, for example in structuretable_paramDef:
<img width="520" alt="image" src="https://user-images.githubusercontent.com/1628100/225784050-8da44b2d-8dfa-434d-aef2-adb0f40c1be4.png">

Take a look at `propertiesController.convertNestedPropertyId()`

Reply:
Take a look at https://github.com/elyra-ai/canvas/blob/main/canvas_modules/common-canvas/src/common-properties/util/control-utils.js#L25.  I think we already have a method that does this.

As long as these id's are unique I don't think we really need to care about them being nested.